### PR TITLE
private chains should not download archive tarballs

### PR DESCRIPTION
**Pull request by nicolasochem**
I tried mkchain with no parameters, and realized that the chain was
trying to download a mainnet snapshot. This is not correct behvior,
fixing.

### CODE REVIEW

File: mkchain/tqchain/mkchain.py [hunk 1]
```diff
@@ -157,9 +157,10 @@ def main():
             "zerotier_network": args.zerotier_network,
             "zerotier_token": args.zerotier_token,
         },
-        # Custom chains should not pull snapshots
+        # Custom chains should not pull snapshots or tarballs
         "full_snapshot_url": None,
         "rolling_snapshot_url": None,
+        "archive_tarball_url": None,
```
Comment (harryttd) [inline mkchain/tqchain/mkchain.py:163]: thanks for catching. THis should also include `rolling_tarball_url`

**Comment by nicolasochem** [comment]
but it's none by default. shall I still add it?

**Comment by harryttd** [comment]
I think it's a good idea to. It definitely doesn't hurt.

**Review by harryttd — APPROVED** [review]
Thanks for handling this.

**Pull request merged**